frontend/packages/e2e/README.md

```diff
@@ -0,0 +1,65 @@
+# e2e
+
+This package is for running end-to-end (E2E) tests using [Playwright](https://playwright.dev/).  
+It allows us to automatically test different parts of our application, including UI changes using **Visual Regression Testing (VRT)**.
+
+## Running Tests Locally
+
+### Run E2E Tests &  Visual Regression Tests
+To execute E2E tests locally, use the following command:
+
+```bash
+pnpm test:e2e
+```
+
+This command runs all end-to-end tests and checks for unexpected UI changes using snapshot comparisons.
+
+### Updating Visual Regression Snapshots
+
+When running VRT for the first time, or when intentionally modifying the UI, you need to update the snapshot images.
+
+For example:
+
+```bash
+URL=http://localhost:5173 pnpm playwright test tests/vrt --update-snapshots
+```
+
+This will update the baseline images used for comparison.
```

Comment:
Q: In this case, `/tests/vrt/vrt.test.ts-snapshots/top-1-chromium-darwin.png` will be added, so can't we use it as is?

I'm thinking that snapshots can only be updated from test results generated by GitHub Actions, is this not the case? 💭

Reply:
In Github Action, we are comparing screenshots of the ``preview`` and ``preview`` screens to each other, and in the local environment, we are comparing screenshots of the ``localhost`` and ``localhost`` screens to each other.

So I feel that if I use the ``localhost`` image in Github Action, I will have to compare the screenshots of ``localhost`` and ``preview``, and it will be difficult to pass the test because of the different environment.🤔💭

Please let me know if I didn't answer your question.

Reply:
OK, I understand. For once, I think it's a good idea to keep it that way 👍🏻 

But even in a local environment, if you use docker and run it in a linux environment, you should get the same results both locally and in preview:

```json
    "_docker": "docker run --rm --ipc=host -v $(pwd):/workspace mcr.microsoft.com/playwright:v$(node -e 'console.log(require(\"./package.json\").devDependencies[\"@playwright/test\"])')-jammy",
    "_test": "playwright test",
    "test:e2e": "pnpm run _docker npm run _test"
```

ref: [PR TIMESにおけるPlaywrightを用いたVisual Regression Test | PR TIMES 開発者ブログ](https://developers.prtimes.jp/2024/04/26/visual-regression-test-using-playwright-in-prtimes/)